### DISCUSSION TITLE
NoConversionRule TimestampTz(true) for Arrow2TypeSystem

### DISCUSSION THREAD

**Discussion by glennpierce** [sfu-db/connector-x#220] [Q&A]
I am trying to load rows from a postgres query into a polars dataframe but getting a conversion error.
                            
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: ConnectorX(NoConversionRule("TimestampTz(true)", "connectorx::destinations::arrow2::typesystem::Arrow2TypeSystem"))'

The query used is

SELECT ts, s1.sensor_id, s1.value
                                FROM sensor_values s1
                                WHERE s1.sensor_id IN (620611,620613,620615,620616,620618,620706) AND s1.ts >= $1 AND
                                        s1.ts < $2
                            ORDER BY 2, 1 OFFSET 0 LIMIT 500000

What can I do to get around this ?

**Comment by wangxiaoying** [comment]
Hi @glennpierce , can you try the newest alpha version: `0.2.4-alpha.6`?

**Reply by glennpierce** [reply]
I tried that and get

thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: PolarsError(InvalidOperation("Cannot create polars series timestamp with timezone"))', libbmos/src/lib.rs:3786:34
